index.html

```diff
@@ -177,53 +187,53 @@ <h3>About Me</h3>
                     don’t hesitate to contact me.</p>
                 <button>Get my Resume</button>
             </div>
-            <div class="about_boxes">
-                <div class="box">
-                    <div> <img src="Assets/icon-languages.png" alt="Languages"></div>
-                    <h3>Languages</h3>
-                    <ul>
-                        <li>JavaScript</li>
-                        <li>Ruby</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                    </ul>
-                </div>
-                <div class="box">
-                    <div> <img src="Assets/icon-frameworks.png" alt="Frameworks"></div>
-                    <h3>Frameworks</h3>
-                    <ul>
-                        <li>React.js</li>
-                        <li>Ruby on Rails</li>
-                        <li>RSpec</li>
-                    </ul>
-                    <ul>
-                        <li>Capybara</li>
-                        <li>Selenium</li>
-                    </ul>
-                </div>
-                <div class="box">
-                    <div> <img src="Assets/icon-skills.png" alt="Skils"></div>
-                    <h3>Skils</h3>
-                    <ul>
-                        <li>Database Management</li>
-                        <li>Version Control</li>
 
-                    </ul>
-                    <ul>
-                        <li>CLI</li>
-                        <li>Web Development</li>
-                        <li>API Design</li>
-                    </ul>
-                </div>
+        </div>
+        <div class="about_boxes">
+            <div class="box">
+                <div> <img src="Assets/icon-languages.png" alt="Languages"></div>
+                <h3>Languages</h3>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="box">
+                <div> <img src="Assets/icon-frameworks.png" alt="Frameworks"></div>
+                <h3>Frameworks</h3>
+                <ul>
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                </ul>
+                <ul>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="box">
+                <div> <img src="Assets/icon-skills.png" alt="Skils"></div>
+                <h3>Skils</h3>
+                <ul>
+                    <li>Database Management</li>
+                    <li>Version Control</li>
+
+                </ul>
+                <ul>
+                    <li>CLI</li>
+                    <li>Web Development</li>
+                    <li>API Design</li>
+                </ul>
             </div>
```

Comment:
You did a good job with the `About me` section, I'd kindly suggest you implement the following changes :

- please add a margin-top property similar to the Figma design so as to have a better UI and show good spacing in your portfolio
- kindly align the background images in  the right position such that one has a higher top value than the other and is not on the same line as seen in the template given 
- please increase the padding and the font size for your button in this section
- please implement the technologies section and boxes to be similar to the design we have in our figma template given

Your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/7fb92509-28d4-41b3-a893-db3457967b0e)

![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/e905e6ea-1b80-41bb-8242-05936b599546)

design given :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/9754e39c-b7a4-4937-a396-36ad35a96b82)

![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/5677f336-cb5c-41cc-9545-6c11a32873be)